multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -254,6 +255,11 @@ def get_column_names(self, modality: str):
                 return self._image_path_names
             else:
                 return self._image_feature_names
+        elif modality == OVD:
+            if hasattr(self, "_image_path_names"):
+                return self._image_path_names + self._text_feature_names
+            else:
+                return self._image_feature_names + self._text_feature_names
```

Comment:
`ovd` is not a modality. I think no need to add this.

Reply:
Later will try to combine with Object Detection to use ROIS modality. But for now since the bounding box need special operation when we support training later, we might need to keep it.